### DISCUSSION TITLE
Colab mode - how to install NVIDIA driver please help

### DISCUSSION THREAD

**Discussion by Mike2k1000** [ehristoforu/DeFooocus#13] [Q&A]
Hello,

I'm trying to run DeeFooocus in Colab mode and it's allways prompting this message:

Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx

Can anybody please help how to solve this.

Thank you.

**Comment by ehristoforu** [comment]
Good afternoon, check whether you have selected a T4 video card.

**Comment by ron9010** [comment]
![2024-10-13_112727](https://github.com/user-attachments/assets/eec22e00-1345-4747-9c1e-95992ce940d1)
what is problem solve? my pc intel grapich? how in possible
